Source/WebKit/Shared/WebGPU/WebGPUPipelineLayoutDescriptor.serialization.in

```diff
@@ -23,5 +23,6 @@
 #if ENABLE(GPU_PROCESS)
 [AdditionalEncoder=StreamConnectionEncoder] struct WebKit::WebGPU::PipelineLayoutDescriptor : WebKit::WebGPU::ObjectDescriptorBase {
     Vector<WebKit::WebGPUIdentifier> bindGroupLayouts;
+    bool isAutoLayout;
```

Comment:
Shouldn't this be represented in-line as an optional?

Reply:
oh make `bindGroupLayouts` an optional?